client/app/controllers/collection/add.js

```diff
@@ -68,99 +68,99 @@ export default Ember.Controller.extend({
         this.saveParameter = this.saveParameter.bind(this);
         this.closeSection = this.closeSection.bind(this);
         this.openSection = this.openSection.bind(this);
-        this.create_widget = this.create_widget.bind(this);
-        this.delete_widget = this.delete_widget.bind(this);
+        this.createWidget = this.createWidget.bind(this);
+        this.deleteWidget = this.deleteWidget.bind(this);
     },
 
     // Fire enabled actions.
     updateState: function(actions) {
         let controller = this;
         actions.forEach((action) => {
             // Check if the action can fire.
-            const may_fire = check_all.call(this, action.conditions);
-            if (!may_fire) return;
+            if (!checkAll.call(this, action.conditions)) return;
```

Comment:
This use of braces is against the advice of airbnb guides.
https://github.com/airbnb/javascript#blocks--braces